executor.go

```diff
@@ -69,25 +75,31 @@ type Executor struct {
 	// not set, the validation result will be returned without evaluation.
 	// Optional.
 	PolicyEnforcer PolicyEnforcer
+
+	// Concurrency limits the maximum number of concurrent execution per
+	// validation request. If less than or equal to 0, a default (currently 3)
+	// is used.
+	Concurrency int
```

Comment:
This is not reflected in `validateExecutorSetup`. In `validateExecutorSetup`, it should somehow detect if `Concurrency <= 0` when calling `ValidateArtifact`. If `Concurrency` is invalid, use `defaultConcurrency` on the fly without modifying the `Concurrency` value.

Reply:
Updated